main.js

```diff
@@ -1,163 +1,553 @@
-// VOORRAAD ARRAY MET TV'S
 const inventory = [
```

Comment:
Regel 1 tot en met 163 kunnen genegeerd worden dit is de array!
Dit komt omdat ik het project al eerder gemaakt had en het voorheen gemaakte project gekopieerd en geplakt had.